samples/cpp/benchmark_app/main.cpp

```diff
@@ -663,42 +711,130 @@ int main(int argc, char* argv[]) {
             }
             ss << niter << " iterations";
         }
+
         next_step(ss.str());
 
+        if (inferenceOnly) {
+            slog::info << "BENCHMARK IS IN INFERENCE ONLY MODE." << slog::endl;
+            slog::info << "Input blobs will be filled once before performance measurements." << slog::endl;
+        } else {
+            slog::info << "BENCHMARK IS IN FULL MODE." << slog::endl;
+            slog::info << "Inputs setup stage will be included in performance measurements." << slog::endl;
+        }
+
+        // copy prepared data straight into inferRequest->getBlob()
+        // for inference only mode
+        if (inferenceOnly) {
+            if (nireq < inputsData.begin()->second.size())
+                slog::warn << "Only " << nireq << " test configs will be used." << slog::endl;
+            size_t i = 0;
+            for (auto& inferRequest : inferRequestsQueue.requests) {
+                auto input = app_inputs_info[i % app_inputs_info.size()];
```

Comment:
Please rename `input` to` inputs`. I'm confusing each time when looking here.
```suggestion
                auto inputs = app_inputs_info[i % app_inputs_info.size()];
```

Reply:
Done